examples/tutorials/analysis-3d/energy_dependence_estimation.py

```diff
@@ -0,0 +1,310 @@
+# extension parameters free. This allows them to be used for fitting the spatial model
+# in each energy band.
+#
+
+model.spatial_model.lon_0.frozen = False
+model.spatial_model.lat_0.frozen = False
+model.spatial_model.sigma.frozen = False
+
+model.spectral_model.amplitude.frozen = False
+model.spectral_model.index.frozen = True
+
+datasets.models = model
+
+estimator = EnergyDependenceEstimator(energy_edges=energy_edges, source="MSH1552")
+
+######################################################################
+# Show the results tables
+# -----------------------
+#
+# The results of the source signal above the background in each energy bin
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Firstly, the estimator is run to produce the results.
+# The table here shows the ∆(TS) value, the number of degrees of freedom (df)
+# and the significance (σ) in each energy bin. The significance values here show that each
+# energy band has significant signal above the background.
+#
+
+results = estimator.run(datasets)
+table_bkg_src = Table(results["src_above_bkg"])
+display(table_bkg_src)
+
+######################################################################
+# The results for testing energy dependence
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Next, the results of the energy-dependent estimator are shown.
+# The table shows the various free parameters for the joint fit for :math:`H_0` across the entire
+# energy band and for each energy bin shown for :math:`H_1`.
+
+ts = results["energy_dependence"]["delta_ts"]
+df = results["energy_dependence"]["df"]
+sigma = ts_to_sigma(ts, df=df)
+
+print(f"The delta_ts for the energy-dependent study: {ts:.3f}.")
+print(f"Converting this to a significance gives: {sigma:.3f} \u03C3")
+
+results_table = Table(results["energy_dependence"]["result"])
+display(results_table)
+
+
+######################################################################
+# The chi-squared value for each parameter of interest
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# We can also utilise the `~gammapy.estimators.energydependence.weighted_chi2_parameter` function for each parameter.
+#
+# The weighted chi-squared significance for the ``sigma`` values.
+#
+
+display(
+    Table(
+        weighted_chi2_parameter(
+            results["energy_dependence"]["result"], parameter="sigma"
+        )
+    )
+)
+
+######################################################################
+# The weighted chi-squared significance for the ``lat_0`` values.
+
+display(
+    Table(
+        weighted_chi2_parameter(
+            results["energy_dependence"]["result"], parameter="lat_0"
+        )
+    )
+)
+
+######################################################################
+# The weighted chi-squared significance for the ``lon_0`` values.
+
+display(
+    Table(
+        weighted_chi2_parameter(
+            results["energy_dependence"]["result"], parameter="lon_0"
```

Comment:
Do you need 3 Tables with one row each? Showing all the parameters in a Table simultaneously will be better

Reply:
The way the `weighted_chi2_parameter` function is created means that one table is created for a given parameter. I will adjust the implementation a bit!